pyproject.toml

```diff
@@ -26,8 +23,8 @@ exclude_dirs = [".venv", "tests", "features"]
 
 [tool.ruff]
 extend-exclude = [
-    "__pycache__",
-    ".vscode*",
+    "[{][{]cookiecutter[.]project_dirname[}][}]/__pycache__",
+    "[{][{]cookiecutter[.]project_dirname[}][}]/.vscode*",
```

Comment:
These rules currently make `ruff format` error out. Confirmed to be a [bug](https://github.com/astral-sh/ruff/issues/9381) on their side.